authenticate/handlers.go

```diff
@@ -61,6 +61,11 @@ func (a *Authenticate) Mount(r *mux.Router) {
 	r.Path("/robots.txt").HandlerFunc(a.RobotsTxt).Methods(http.MethodGet)
 	// Identity Provider (IdP) endpoints
 	r.Path("/oauth2/callback").Handler(httputil.HandlerFunc(a.OAuthCallback)).Methods(http.MethodGet)
+	s := r.PathPrefix("/oauth2/session").Subrouter()
+	s.Use(func(h http.Handler) http.Handler {
+		return sessions.RetrieveSession(a.state.Load().sessionLoaders...)(h)
+	})
+	s.Path("/frontchannel-logout").Handler(httputil.HandlerFunc(a.FrontchannelLogout)).Methods(http.MethodGet)
```

Comment:
Does the URL have to be under oauth2/session? From the spec, it looks to be unspecified. If so, could we put this under `.pomerium` ?

Reply:
Also, it probably makes sense to add this endpoint to the wellKnown response handler? 

```
  "frontchannel_logout_session_supported": true
```

And perhaps a https://ldapwiki.com/wiki/Frontchannel_logout_uri

Reply:
I added `frontchannel_logout_uri` to the wellknown handler as I think this may help someone to find out the correct url to register. But I don't understand how `"frontchannel_logout_session_supported": true` would help anybody?
  
Maybe we should document this feature in the Docs/Guides, could you give me a hint where it fits?

Reply:
> added frontchannel_logout_uri to the wellknown handler as I think this may help someone to find out the correct url to register. 

Agree. 

> But I don't understand how "frontchannel_logout_session_supported": true would help anybody?

I'm with you :) 

> Maybe we should document this feature in the Docs/Guides, could you give me a hint where it fits?

That would be great. I feel like this would fit well in `docs/docs/topics` and maybe have an article called "logging a user out" or "end a user's session" or something like that. 

Thoughts?

Reply:
> Thoughts?

I named the topic "Single Sign-out" as I think it describes it better than "logging out" which already is handled by the sign-out route